### DISCUSSION TITLE
fromitive-blog/coding-interview/2023-07-10-hackerrank-1/

### DISCUSSION THREAD

**Discussion by giscus** [fromitive/fromitive-blog#8] [Q&A]
# fromitive-blog/coding-interview/2023-07-10-hackerrank-1/

HackerRank 문제 Hackerland Radio Transmitters 리뷰 (미해결)

https://fromitive.github.io/fromitive-blog/coding-interview/2023-07-10-hackerrank-1/

**Comment by DDONONG** [comment]
쉬었다가 다시 보면 풀 수 있을거예요!
